### PR TITLE
Implemented story postcards

### PR DESCRIPTION
**Pull request by partysnax**
#134 

The "Share" button for each card does not have a function yet.

### CODE REVIEW

**Comment by partysnax** [comment]
Samples:
2021-03-27_17h39_37
2021-03-27_17h40_18
2021-03-27_17h40_39

File: partials/story-postcard.hbs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                <p class="text-sm my-1">
+                    {{#if primary_tag}}
+                    {{primary_tag.name}} - 
+                    {{else}}
+                    No tag - 
+                    {{/if}}
+                    <time datetime="{{date format="YYYY-MM-DD"}}">{{date published_at format="D MMMM YYYY"}}</time>
+                </p>
+                <p class="text-lg font-bold my-2">{{title}}</p>
+                <p class="text-sm my-1 max-h-20 overflow-hidden">{{excerpt}}</p>
```
Comment (wei2912) [inline partials/story-postcard.hbs:20]: Could we use `overflow-ellipsis` instead?

File: partials/story-postcard.hbs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        {{/if}}
+        <div class="flex-grow bg-gray-100 px-3 pt-2 pb-4">
+            <div>
+                <p class="text-sm my-1">
+                    {{#if primary_tag}}
+                    {{primary_tag.name}} - 
+                    {{else}}
+                    No tag - 
+                    {{/if}}
+                    <time datetime="{{date format="YYYY-MM-DD"}}">{{date published_at format="D MMMM YYYY"}}</time>
```
Comment (wei2912) [inline partials/story-postcard.hbs:17]: I think it's okay to use a shorter month format here, e.g. `4 Jan 2021`.

File: partials/story-postcard.hbs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        {{#if feature_image}}
+        <img class="object-cover m-0 w-full h-60 md:w-2/5 md:h-full lg:w-full lg:h-60" src="{{img_url feature_image}}" alt="{{title}}" />
+        {{else}} {{!-- Placeholder --}}
+        <img class="object-cover m-0 hidden md:block md:w-2/5 md:h-full lg:w-full lg:h-60" src="https://via.placeholder.com/100/FFFFFF/FFFFFF" alt="{{title}}" />
+        {{/if}}
+        <div class="flex-grow bg-gray-100 px-3 pt-2 pb-4">
+            <div>
+                <p class="text-sm my-1">
+                    {{#if primary_tag}}
+                    {{primary_tag.name}} -
```
Comment (wei2912) [inline partials/story-postcard.hbs:13]: Preferably the tag should be displayed underneath the title in the form of pills, something like https://material-ui.com/components/chips/. But I think we can do this in a separate PR.

**Review by wei2912 — CHANGES_REQUESTED** [review]
Looks good overall, I think it's nearly ready to be merged. Do create a separate issue for the tags as requested in my comment below, once you make the requested changes.

File: partials/story-postcard.hbs [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{{! prettier-ignore }}
```
Comment (wei2912) [inline partials/story-postcard.hbs:1]: To be removed.

File: partials/story-postcard.hbs (hunk 1 above)
Comment (partysnax) [reply partials/story-postcard.hbs:20]: Unfortunately `overflow-ellipsis` doesn't work when the text has to be wrapped around multiple lines. 
Without the `whitespace-nowrap` style, the ellipsis doesn't show at all.
With the above style: 
2021-06-13_16h39_35

Comment (wei2912) [reply partials/story-postcard.hbs:19]: Hmm alright, never mind then... we can look into a better solution for this.

**Review by wei2912 — APPROVED** [review]
